leapp/reporting/__init__.py

```diff
@@ -3,6 +3,7 @@
 from leapp.models import fields, Model
 from leapp.topics import ReportTopic
 from leapp.libraries.stdlib.api import produce
+from leapp.compat import string_types
```

Comment:
Please rearrange imports in alphabetical order. 

Hmm, wondering why didn't linter catch that?